mixins/visible-on-ancestor/visible-on-ancestor-mixin.js

```diff
@@ -134,7 +150,7 @@ export const VisibleOnAncestorMixin = superclass => class extends superclass {
 	}
 
 	__voaShow() {
-		if (reduceMotion) {
+		if (reduceMotion && this.animationType !== 'opacity') {
```

Comment:
For our usecase, we want the opacity transition to still run when prefers-reduce-motion (otherwise it gets too flicker-y). I realize that this isn't standard, so let me know other thoughts around this.

Reply:
Is this flickeryness an issue in other places we use VOA and animations are turned off?

Reply:
Hmmm, in the `handleTransitionEnd` below, it looks like there may be an assumption made, that `transform` will always be a property that transitions to end. If you haven't already, it might be good to step through this handler or add console logs to make sure it's running as it should.

If we are still transitioning `opacity` (but not `transform`) when reduced motion is turned on, then that check might need to be a little different. Maybe checking `transform` or `opacity` would be appropriate.

Reply:
The main other use case seems to be in cards. In that case it seems fine, as the cards are spacious enough that going quickly between them the dropdown appearing and disappearing doesn't seem too jarring. It's more of an issue here because of the list usage where the add buttons would be close together. However, it is totally possible for other places to be flickery too.